doc/release-notes/release-8.sgml.in

```diff
@@ -36,12 +36,11 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 <sect1>Helper changes
 <p>
 <descrip>
-	<p>Removed <em>basic_smb_lm_auth</em> NTLM authentication helper.
-	   Use the <em>ntlm_auth</em> helper from the Samba project instead.
-
 	<p>Removed <em>ext_lm_group_acl</em> external acl helper. No product
 	   currently on the market suports the LanManager protocol
 	   in their default configuration.
+	<p>Removed <em>basic_smb_lm_auth</em> NTLM authentication helper.
```

Comment:
alphabetical order please so people can find these easily when/if the list gets long.

Reply:
re-sorted